src/au3audio/audioenginetypes.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Audacity: A Digital Audio Editor
+ */
+#pragma once
+
+#include "global/types/string.h"
+#include <optional>
+
+namespace au::audio {
+using RtEffectState = void*;
+
+using EffectId = muse::String;
+
+enum class RtEffectListEventType
+{
+    Added,
+    Removed,
+    Replaced,
+};
+
+using TrackId = long;
+
+struct RtEffectListItem {
```

Comment:
better not to use the abbreviation, at first I didn't understand what rt is

Reply:
I am also confused by the Item in the audio module. Item is about UI, as for me, the audio module should not operate with such concepts

Reply:
I already have a `RealtimeEffectListItem.qml` :) Wanted to avoid the two getting confused.
I agree that "list item" does sound UI-like indeed. However, the sequence of realtime effects we set on the audio engine (or in the effect module, debate is ongoing) needs a name. What do you suggest ? "Sequence" instead of "List" would do. "Entry" instead of "Item", too. Pick your combination :)